src/execution/__tests__/subscribe-test.ts

```diff
@@ -421,6 +421,48 @@ describe('Subscription Initialization Phase', () => {
     expect(() => subscribe({ schema })).to.throw('Must provide document.');
   });
 
+  it('allows positional arguments to createSourceEventStream', () => {
```

Comment:
Please switch all tests to use named argument, and make this a test to support positional arguments.

Reply:
Hmmm, I may need more coffee, but I think that's done already. There are very few direct tests of `createSourceEventStream`, as the code is mostly tested through calls to `subscribe`.

Reply:
wow, we use it only in one place in whole tests.
When totally make sense.
Just got confused because I expected more changes.

When everything is good, just small improvement to the description
```suggestion
  it('Deprecated: allows positional arguments to createSourceEventStream', () => {
```

Similar to this one
https://github.com/graphql/graphql-js/blob/75635f03ba950a5f0aa212bad2052297e9a59725/src/validation/__tests__/validation-test.ts#L61

Reply:
Done!